### PR TITLE
FIX New default tts missing

### PR DESCRIPTION
**Pull request by provinzio**
Closes https://github.com/custom-components/alexa_media_player/issues/1764

### CODE REVIEW

File: custom_components/alexa_media/notify.py [hunk 1]
```diff
@@ -212,6 +212,7 @@ async def async_send_message(self, message="", **kwargs):
         targets = kwargs.get(ATTR_TARGET)
         title = kwargs.get(ATTR_TITLE, ATTR_TITLE_DEFAULT)
         data = kwargs.get(ATTR_DATA, {})
+        data = data if data is None else {}
```
Comment (alandtse) [inline custom_components/alexa_media/notify.py]: Isn't this flipped?

Comment (provinzio) [reply custom_components/alexa_media/notify.py]: Stupid last minute change 😴 good eye. I'll adjust it.

**Review by alandtse — APPROVED** [review]
Thanks

**Comment by pedolsky** [comment]
Beautiful. Thank you both!

**Pull request merged**